pkg/datamodel-tests/src/implementations/version-1.1/note.ts

```diff
@@ -0,0 +1,105 @@
+import { VolumeStock, VolumeTransactionTuple } from '@/types';
+import { sortBooks } from '@/utils/misc';
+import { NoteInterface, WarehouseInterface, NoteData } from './types';
+
+export class Note implements NoteInterface {
```

Comment:
Should we export the class? It means that code importing `Note` needs to use `new`, which might be unexpected. I think it might be good to consider this an implementation detail, and use `newNote` as the public API.

Reply:
Makes sense...I completely missed that, and I had the `newNote` function to begin with...the export is completely unnecessary.